bosh-stemcell/spec/stemcells/ubuntu_trusty_spec.rb

```diff
@@ -177,6 +184,24 @@
     end
   end
 
+  context 'installed by bosh_softlayer_agent_settings', {
+      exclude_on_aws: true,
+      exclude_on_vcloud: true,
+      exclude_on_vsphere: true,
+      exclude_on_warden: true,
+      exclude_on_azure: true,
+      exclude_on_openstack: true,
+  } do
+    describe file('/var/vcap/bosh/agent.json') do
+      it { should be_valid_json_file }
+      it { should contain('"Type": "File"') }
+      it { should contain('"MetaDataPath": ""') }
+      it { should contain('"UserDataPath": "/var/vcap/bosh/user_data.json"') }
```

Comment:
@cppforlife  I don`t know why this case will fail, in fact, I could generate the stemcell by using bundle exec rake stemcell:build[softlayer,esxi,ubuntu,trusty,go,bosh-os-images,bosh-ubuntu-trusty-os-image.tgz] with no problem